jsapp/js/components/processing/analysis/analysisQuestions.reducer.ts

```diff
@@ -6,6 +6,12 @@ import {applyUpdateResponseToInternalQuestions} from './utils';
 export interface AnalysisQuestionsState {
   /** Whether any async action is being done right now. */
   isPending: boolean;
+  /**
+   * True if user has made a change and the change has not been requested OR
+   * backend differs from frontend. This is determined by a 200 response or the
+   * pending state.
+   */
```

Comment:
I think this would be more precise:

> It's `true` if user has made a change in the UI and the Back end has not been updated yet. Every completed call to API will change it back to `false`.